geniza/footnotes/models.py

```diff
@@ -335,6 +339,10 @@ def formatted_display(self, extra_fields=True):
     all_authors.short_description = "Authors"
     all_authors.admin_order_field = "first_author"  # set in admin queryset
 
+    def formatted_stripped(self):
+        """strip HTML tags and trailing period from formatted display"""
+        return strip_tags(self.formatted_display(extra_fields=False))[:-1]
```

Comment:
seems like it would be more semantic and less brittle to use `.rstrip(".")`

why do we strip the trailing period from this version?

Reply:
We'd stripped it for a purpose will be made obsolete by #551 😆  (using a trailing comma before footnote location in doc detail page). Removed in af1b073